front/app/components/admin/PostManager/index.tsx

```diff
@@ -492,7 +489,15 @@ const Data = adopt<DataProps, InputProps>({
     }
 
     if (type === 'ProjectIdeas' && projectId) {
-      return <GetTopics projectId={projectId}>{render}</GetTopics>;
+      return (
+        <GetProjectAllowedInputTopics projectId={projectId}>
```

Comment:
Do we have a hook for this? Perhaps it's better to use it/ create it as we are slowly trying to get rid of these resource components.